public/core.d.ts

```diff
@@ -10987,6 +10987,122 @@ declare namespace JMap {
       function getCopyWithoutDuplicate(array: Array<string | number>): Array<string | number>
     }
 
+    /**
+     * **JMap.Util.Ajax**
+     * 
+     * Here you'll find all JMap Ajax related methods
+     */
+    namespace Ajax {
+
+      /**
+       * ** JMap.Util.Ajax.get
+       * 
+       * Used to retreive data from the server at the specified resource.
```

Comment:
pour toutes les description, utiliser "Retrieves" (Inserts, Deletes, etc ....) au lieu de "Used to retreive" (typo dans "retreive"), et mettre un retour de chariot entre la description et les params